doc/programming_guide/sound.rst

```diff
@@ -0,0 +1,724 @@
+    # a pathlib.Path,
+    pathlib_sound = arcade.load_sound(Path("imaginary\windows\path\file.wav"))
+    # or an ordinary string describing a path.
+    string_path_sound = arcade.load_sound("imaginary/mac/style/path.wav")
+
+If you prefer a more object-oriented style, you can create
+:py:class:`~arcade.Sound` instances directly::
+
+    from arcade import Sound  # You can also use arcade.Sound directly
+
+    # Although Sound accepts the same arguments as load_sound,
+    # only the built-in resource handle is shown here.
+    hurt_sound = Sound(":resources:sounds/hurt1.wav")
+
+See the following to learn more:
+
+#. :ref:`Platformer Part 7 - Loading Sounds <platformer_part_seven_loading_sounds>`
+#. :ref:`resources`
+#. :py:mod:`pathlib`
+#. :ref:`sound-loading-modes`
+
+.. _sound-basics-playing:
+
+Playing Sounds
+^^^^^^^^^^^^^^
+
+There are two easy ways to play a :py:class:`~arcade.Sound` object.
+
+One is to call :py:meth:`Sound.play <arcade.Sound.play>` directly::
+
+    self.hurt_player = hurt_sound.play()
+
+The other is to pass a :py:class:`~arcade.Sound` instance as the first
+argument of :py:func:`arcade.play_sound`::
+
+    # Important: this *must* be a Sound instance, not a path or string!
+    self.hurt_player = arcade.play_sound(hurt_sound)
+
+Both return a :py:class:`pyglet.media.player.Player`. You should store
+it somewhere if you want to be able to stop or alter a specific playback of
+a :py:class:`~arcade.Sound`'s data.
+
+Sounds vs pyglet Players
+""""""""""""""""""""""""
+This is a very important distinction:
+
+* An :py:class:`arcade.Sound` represents a source of audio data
+* Arcade uses pyglet's :py:class:`~pyglet.media.player.Player` to
+  represent a specific playback of audio data
+
+Imagine you have two characters in a game which both play the same
+:py:class:`~arcade.Sound` when moving. Since they are separate
+characters in the world, they have separate playbacks of that sound.
+This means each stores its own :py:class:`~pyglet.media.player.Player`.
+
+The separate pyglet players allow controlling their playbacks of the
+movement sound separately. For example, one character may get close
+enough to the user's character to talk, attack, or perform some other
+action. You would use that character's specific pyglet
+:py:class:`~pyglet.media.player.Player` to stop the corresponding
+playback of the movement sound.
+
+This is crucial for games which hide parts of the world from view.
+An enemy with no way for the user to know it's there is the most
+common version of the unknown danger mentioned in :ref:`sound-why-important`.
+
+See the following to learn more:
+
+#. :ref:`Platformer Tutorial - Part 7 - Collision Detection <platformer_part_seven_playing_sounds>`
+#. :ref:`sound_demo`
+
+.. _sound-basics-stopping:
+
+Stopping Sounds
+^^^^^^^^^^^^^^^
+
+
+Arcade's helper functions are the easiest way to stop playback. To use them, do
+the following:
+
+#. Choose one of the following:
```

Comment:
Having 'following' in here twice is a bit confusing. Any way we could tighten this language?